src/apps/lwaftr/address_map.lua

```diff
@@ -97,7 +100,7 @@ local function attach_lookup_helper(map)
 
    function map:lookup_psid(ipv4, port)
       local psid_info = self:lookup(ipv4).value
-      local psid_len, shift = psid_info.psid_len, psid_info.shift
+      local psid_len, shift = psid_info.psid_length, psid_info.shift
```

Comment:
Nit: Since the name of the field is now `psid_length` I would use the same name for the local variable.